clang/Util.d

```diff
@@ -80,8 +80,17 @@ mixin template CX ()
 }
 
 extern (C) int mkstemps(char*, int);
+extern (C) char* mkdtemp(char*);
 extern (C) int close(int);
 
+private void randstr (char[] slice)
+{
+    import std.random;
+
+    for (uint i = 0; i < slice.length; ++i)
```

Comment:
`foreach`? Or is it even possible to use array operations, something like: `slice[] = uniform!("[]")('A', 'Z');`, and skip the loop completely.